### ISSUE TITLE
Aria Labels

### ISSUE THREAD

**Issue by brianjgeiger** [calvinlough/ember-tag-input#15]
For accessibility, all input fields need labels of some kind. Our design doesn't work with a wrapping label, so it would be awesome to have the ability to use an aria label option in the control.

**Comment by brianjgeiger** [comment]
Apparently, as I discovered working on the PR for this, the big problem is that Ember's input helper doesn't support aria-label. So there's not a straightforward way to do this.
